bind_paths_and_mounts.rst

```diff
@@ -27,16 +27,27 @@ system-defined bind paths and user-defined bind paths.
 System-defined bind paths
 *************************
 
-The system administrator has the ability to define what bind paths will
-be included automatically inside each container. Some bind paths are
-automatically derived (e.g. a user’s home directory) and some are
-statically defined (e.g. bind paths in the {Singularity} configuration
-file). In the default configuration, the system default bind points are
-``$HOME`` , ``/sys:/sys`` , ``/proc:/proc``, ``/tmp:/tmp``,
-``/var/tmp:/var/tmp``, ``/etc/resolv.conf:/etc/resolv.conf``,
-``/etc/passwd:/etc/passwd``, and ``$PWD``. Where the first path before
-``:`` is the path from the host and the second path is the path in the
-container.
+The system administrator has the ability to define which bind paths will be
+included automatically inside each container. Some bind paths are derived (e.g.
+a user's home directory), and some are statically defined with a ``bind path``
+entry in ``singularity.conf``.
+
+In the default configuration, the default bind mounts are:
+
+- The user's home directory (``$HOME``)
+- The current working directory (``CWD``), unless its path contains symlinks
+  resolving to different locations on the host vs inside the container.
+- ``/dev``
+- ``/etc/hosts``
+- ``/etc/localtime`` 
+- ``/proc``
+- ``/sys``
+- ``/tmp``
+- ``/var/tmp``
+
+Additionally, customised system configuration files ``/etc/resolv.conf`` and
```

Comment:
Conform to our American chauvinism and spell customi[z]ed with a... zed? 😁

(More seriously, this is your call, I just wanted to make sure this wasn't unintentional.)